client/src/index.jsx

```diff
@@ -1,12 +1,163 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
+import './styles.css';
+
+class App extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {number: 1};
+		this.addSighting = this.addSighting.bind(this);
+	}
+
+	renderSightings(){
+		const {number} = this.state;
+		if (!number | number < 0) return null;
+		let total = []
```

Comment:
Why is this a `let`?

Reply:
Since we push an unknown number of times in the next two lines, I thought const wouldn't be appropriate. What would you suggest?

Reply:
`const` is fine. const means constant reference, not a constant